bugsnag-plugin-android-exitinfo/src/main/java/com/bugsnag/android/TombstoneParser.kt

```diff
@@ -3,20 +3,53 @@ package com.bugsnag.android
 import android.app.ApplicationExitInfo
 import android.os.Build
 import androidx.annotation.RequiresApi
+import com.bugsnag.android.Thread.State
+import com.bugsnag.android.repackaged.server.os.TombstoneProtos
 import com.bugsnag.android.repackaged.server.os.TombstoneProtos.Tombstone
+import com.bugsnag.android.Thread as BugsnagThread
 
 internal class TombstoneParser(
     private val logger: Logger
 ) {
+
     @RequiresApi(Build.VERSION_CODES.R)
-    fun parse(exitInfo: ApplicationExitInfo) {
+    fun parse(
+        exitInfo: ApplicationExitInfo,
+        threadConsumer: (BugsnagThread) -> Unit
+    ) {
         try {
             val trace: Tombstone = exitInfo.traceInputStream?.use {
                 Tombstone.newBuilder().mergeFrom(it).build()
             } ?: return
-            trace.threadsMap.values
+            convertToBugsnagThread(trace.threadsMap.values, threadConsumer)
         } catch (ex: Throwable) {
             logger.w("Tombstone input stream threw an Exception", ex)
         }
     }
+
+    private fun convertToBugsnagThread(
+        values: MutableCollection<TombstoneProtos.Thread>,
+        threadConsumer: (BugsnagThread) -> Unit
+    ) {
+        values.forEach { thread ->
+            val stackFrames = thread.currentBacktraceList.map {
```

Comment:
These lambdas feel like they're overusing `it`, I think the code would be much clearer if these
threads were named?
```suggestion
            val stackFrames = thread.currentBacktraceList.map { thread ->
```

Reply:
I change it to `tombstoneTraceFrame` since it can be different from the thread from values